specs/airr-schema.yaml

```diff
@@ -2259,118 +2253,152 @@ Clone:
         clone_id:
             type: string
             description: Identifier for the clone.
+            x-airr:
+                miairr: false
+                required: true
         data_processing_id:
             type: string
             description: Identifier for the data processing object.
+            x-airr:
+                miairr: false
         v_call:
             type: string
             description: V gene with allele. For example, IGHV4-59*01.
             example: IGHV4-59*01
+            x-airr:
+                miairr: false
+                required: true
         d_call:
             type: string
             description: D gene with allele. For example, IGHD3-10*01.
             example: IGHD3-10*01
+            x-airr:
+                miairr: false
+                required: true
         j_call:
             type: string
             description: J gene with allele. For example, IGHJ4*02.
             example: IGHJ4*02
+            x-airr:
+                miairr: false
+                required: true
         junction:
             type: string
             description: >
                 Junction region nucleotide sequence, where the junction is defined as
                 the CDR3 plus the two flanking conserved codons.
+            x-airr:
+                miairr: false
         junction_aa:
             type: string
             description: >
                 Junction region amino acid sequence.
+            x-airr:
+                miairr: false
         junction_length:
             type: integer
             description: Number of nucleotides in the junction sequence.
+            x-airr:
+                miairr: false
         junction_aa_length:
             type: integer
             description: Number of amino acids in the junction sequence.
+            x-airr:
+                miairr: false
         germline_alignment:
             type: string
             description: >
                 Assembled, aligned, full-length inferred germline sequence spanning the same region
                 as the sequence_alignment field (typically the V(D)J region) and including the same set
                 of corrections and spacers (if any).
+            x-airr:
+                miairr: false
         germline_alignment_aa:
             type: string
             description: >
                 Amino acid translation of the assembled germline sequence.
+            x-airr:
+                miairr: false
         v_alignment_start:
             type: integer
             description: >
                 Start position in the V segment in both the sequence_alignment and germline_alignment fields
                 (1-based closed interval).
```

Comment:
Same type of comment on these fields, too, I guess, There is no `sequence_alignment` field in this object, so at least the description has to change. I think it would make more sense to just have the fields be named `v_start`, `v_end`, etc, since with only one sequence (the inferred naive ancestor) there isn't really an alignment...

Reply:
Does this still need addressing, @scharch?

Reply:
nope all good